### PR TITLE
Added basic Slovacian reader localization

### PR DESCRIPTION
**Pull request by ojsbsb**
@mtub In one of our current projects, we have translated the basic keys for the reader interface into Slovacian. Everything other than what readers will see is not translated, though, as the editorial workflow is not used in Slovacian.

### CODE REVIEW

File: locale/sk_SK/manager.xml [hunk 1]
```diff
@@ -0,0 +1,741 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/en_US/manager.xml
```
Comment (mtub) [inline locale/sk_SK/manager.xml]: wrong path -> locale/sk_SK/manager.xml

File: locale/sk_SK/locale.xml [hunk 2]
```diff
@@ -0,0 +1,1187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/en_US/locale.xml
```
Comment (mtub) [inline locale/sk_SK/locale.xml]: wrong path

File: locale/sk_SK/editor.xml [hunk 3]
```diff
@@ -0,0 +1,222 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * locale/en_US/editor.xml
```
Comment (mtub) [inline locale/sk_SK/editor.xml]: wrong path

File: plugins/blocks/fontSize/locale/sk_SK/locale.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/blocks/fontSize/locale/en_US/locale.xml
```
Comment (mtub) [inline plugins/blocks/fontSize/locale/sk_SK/locale.xml]: wrong path

please check other plugins, too

**Comment by mtub** [comment]
Thanks, @ojsbsb. This looks good. Please re-check the path names in the header comments (see my other comments). 

Furthermore, I recommend removing the untranslated parts from the files. This makes tracking them in the future easier, and the missing parts will be shown by the translator plugin's check routine.

**Comment by ojsbsb** [comment]
Updated the locale paths, sorry we missed these. 

We are now deleting the untranslated lines. Seems we accidentally looked at exactly these few files which contained both translated and untranslated lines like https://github.com/pkp/ojs/blob/ojs-dev-2_4/locale/sr_SR/manager.xml.

**Comment by ojsbsb** [comment]
Done, @mtub.

**Comment by mtub** [comment]
Thanks @ojsbsb ! Will merge now. Will also look at the sr_SR code you've linked to.

**Pull request merged**